service/shard.go

```diff
@@ -135,7 +135,9 @@ func (sp ShardProxies) ProxyForRequest(r *http.Request) (*httputil.ReverseProxy,
 	// parse height from the request
 	parsedHeight, err := decode.ParseBlockNumberFromParams(decodedReq.Method, decodedReq.Params)
 	if err != nil {
-		sp.Error().Msg(fmt.Sprintf("expected but failed to parse block number for %+v: %s", decodedReq, err))
+		if err != decode.ErrUncachaebleByBlockNumberEthRequest {
+			sp.Error().Msg(fmt.Sprintf("expected but failed to parse block number for %+v: %s", decodedReq, err))
+		}
```

Comment:
great addition here 🙏 logs will be so much more meaningful now

Reply:
I think there are more places with spamming logs, but let's see after redeploy